pkg/harvester/l10n/en-us.yaml

```
@@ -344,6 +344,7 @@ harvester:
       manufacturer: Manufacturer
       serialNumber: Serial Number
       model: Model
+      etcd: Etcd Node
```

```suggestion
      etcd: etcd Node
```

lower case for convention.

---

@torchiaf please hold, we are discussing if we should move the term to "Witness node". It makes more sense since an ordinary controller node can contain etcd too. I'll update again.

---

Please use the term "Witness node" here, thanks!

---

Updated, thanks!